layouts/partials/widgets/about.html

```diff
@@ -1,4 +1,5 @@
 <!-- About/Biography widget -->
+{{ with .page }}
```

Comment:
Please use `{{ $page := .page }}` instead and make the necessary changes to reflect this. Then it's more consistent with the other widget templates and avoids any potential confusion with nesting `with` statements.